.github/workflows/ci.yml

```diff
@@ -0,0 +1,18 @@
+name: CI
+on: [push, pull_request]
```

Comment:
I think it would be better to run the CI only on the PR's

Reply:
If we do it that way I don't the the tests trigger when adding fixes to a PR. Does it?

We can test and see if you'd like.